lib/compress/zstd_opt.c

```diff
@@ -265,6 +264,147 @@ static U32 ZSTD_insertAndFindFirstIndexHash3 (ZSTD_CCtx* const cctx, const BYTE*
 /*-*************************************
 *  Binary Tree search
 ***************************************/
+/** ZSTD_insertBt1() : add one or multiple positions to tree.
+ *  ip : assumed <= iend-8 .
+ * @return : nb of positions added */
+static U32 ZSTD_insertBt1(ZSTD_CCtx* zc,
```

Comment:
Just to make sure, this is just copied from the old `ZSTD_lazy.c` with no other changes, right?

Reply:
Yes. Since `ZSTD_insertBt1()` is no longer provided by `zstd_lazy.c`, it needs to be implemented directly within `zstd_opt.c`.